source/install/native.rst

```diff
@@ -153,4 +153,4 @@ command and look for the line that starts with "active environment":
    conda info
 
 .. _`Miniconda`: https://docs.conda.io/en/latest/miniconda.html
-.. _`Miniconda instructions`: https://conda.io/projects/conda/en/latest/user-guide/install/index.html
\ No newline at end of file
+.. _`Miniconda instructions`: https://conda.io/projects/conda/en/latest/user-guide/install/index.html
```

Comment:
What changed on this line?

Reply:
Sorry about the noop. Looks like VSCode might have added a newline to the end of the file, and I missed it in review.

![image](https://user-images.githubusercontent.com/39198770/83058366-1bd22f80-a00d-11ea-8e97-deb13ec8de90.png)

Reply:
Looks like this still unresolved, can you please undo this part of the changeset?